ci/stemcell/light/pipeline.yml

```diff
@@ -2,72 +2,142 @@
 groups:
   - name: light-gce-stemcells
     jobs:
-      - ubuntu-trusty-stemcell
+      - ubuntu-trusty-stemcell-latest
+      - ubuntu-trusty-stemcell-3312
+      - ubuntu-trusty-stemcell-3263
+
+shared:
```

Comment:
I haven't seen the `shared` key before. Is this something new(er) in Concourse? If so, maybe we can update the CI readme to reflect the minimum Concourse version.